wine-{net, sc, start, uninstaller}, winepath: add page (#22991)
tldr-pages/tldr: pages/linux/winepath.md

# winepath

> Convert file paths between Windows and Unix formats for a Wine prefix.
> More information: <https://manned.org/winepath>.

> - Convert a Windows path to a [u]nix path:

Reviewer: ```suggestion
- Convert a Windows path to a unix path:
```

> `winepath -u '{{C:\path\to\file}}'`

Reviewer: ```suggestion
`winepath {{[-u|--unix]}} '{{C:\path\to\file}}'`
```
Do not use shortform by themselves when logform is available.

> - Convert a Unix path to a long [w]indows path:
> 
> `winepath -w {{path/to/file}}`

Reviewer: ```suggestion
- Convert a Unix path to a long windows path:

`winepath {{[-w|--windows]}} {{path/to/file}}`
```

> - Convert a short (8.3) Windows path of an existing file to its [l]ong form:
> 
> `winepath -l '{{C:\PROGRA~1}}'`

Reviewer: ```suggestion
- Convert a short (8.3) Windows path of an existing file to its long form:

`winepath {{[-l|--long]}} '{{C:\PROGRA~1}}'`
```

> - Convert a long Windows path of an existing file to its [s]hort (8.3) form:
> 
> `winepath -s '{{C:\Program Files}}'`

Reviewer: ```suggestion
- Convert a long Windows path of an existing file to its short (8.3) form:

`winepath {{[-s|--short]}} '{{C:\Program Files}}'`
```

- Separate output paths with a null character instead of a newline:

> `winepath -u -0 {{path1 path2 ...}}`

Reviewer: ```suggestion
`winepath {{[-u|--unix]}} -0 {{path1 path2 ...}}`
```

- Display help:

> `winepath --help`

Reviewer: ```suggestion
`winepath {{[-h|--help]}}`
```
